### PR TITLE
Add icons to some firebrands spells

### PR DESCRIPTION
**Pull request by InfamousSky**

### CODE REVIEW

File: packs/data/equipment.db/the-vision.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
         "baseItem": null,
         "containerId": null,
         "description": {
+            "gm": "<p><strong>Epitome</strong> Harrowheart&rsquo;s workshops become enhanced. Any character may now use a workshop in Harrowheart to attempt to Borrow a Spell, Identify Alchemy, Identify Magic, or Learn a Spell. If they do so, they apply the workshop&rsquo;s item bonus to their skill check.&nbsp;</p>",
```
Comment (simonward) [inline packs/data/equipment.db/the-vision.json:9]: These probably don’t need to be escaped.

File: packs/data/stolen-fate-bestiary.db/endlo-kiver.json [hunk 2]
```diff
@@ -3644,7 +3644,8 @@
                         "domain": "all",
                         "key": "RollOption",
                         "option": "resolute-stance",
-                        "toggleable": true
+                        "toggleable": true,
+                        "value": false
```
Comment (simonward) [inline packs/data/stolen-fate-bestiary.db/endlo-kiver.json:3648]: Don’t need “value”: false

File: packs/data/stolen-fate-bestiary.db/raskus.json [hunk 3]
```diff
@@ -1095,11 +1095,20 @@
...
+                                    "opportunistic-attack"
                                 ]
                             }
                         ],
                         "selector": "strike-damage"
+                    },
+                    {
+                        "domain": "damage",
+                        "key": "RollOption",
+                        "label": "Target has reduction to Speed",
```
Comment (simonward) [inline packs/data/stolen-fate-bestiary.db/raskus.json:1108]: Consider localizing the label.

File: packs/data/equipment.db/the-vision.json (hunk 1 above)
Comment (TikaelSol) [inline packs/data/equipment.db/the-vision.json:9]:
```suggestion
            "gm": "<p><strong>Epitome</strong> Harrowheart's workshops become enhanced. Any character may now use a workshop in Harrowheart to attempt to Borrow a Spell, Identify Alchemy, Identify Magic, or Learn a Spell. If they do so, they apply the workshop's item bonus to their skill check.</p>",
```

File: packs/data/stolen-fate-bestiary.db/raskus.json (hunk 3 above)
Comment (TikaelSol) [reply packs/data/stolen-fate-bestiary.db/raskus.json:1108]: We can localize later, need this in now.

**Pull request merged**